MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'underscore',
+    'uiComponent',
+    'Magento_MediaGalleryUi/js/action/getDetails'
+], function ($, _, Component, getDetails) {
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            template: 'Magento_MediaGalleryUi/image/image-edit',
+            modalSelector: '',
+            imageEditDetailsUrl: '/media_gallery/image/details',
+            saveDetailsUrl: '/media_gallery/image/saveDetails',
+            images: [],
+            tagListLimit: 7,
+            showAllTags: false,
```

Comment:
I think `tagListLimit` and `showAllTags` properties are not applicable to the edit slide panel and can be removed